AUTHORS.md

```diff
@@ -25,13 +25,13 @@ The following people have made contributions to this project:
 - [Adam Dybbroe (adybbroe)](https://github.com/adybbroe)
 - [Ulrik Egede (egede)](https://github.com/egede)
 - [Joleen Feltz (joleenf)](https://github.com/joleenf)
-- [Stephan Finkensieper (sfinkens)](https://github.com/sfinkens)
+- [Stephan Finkensieper (sfinkens)](https://github.com/sfinkens) - Deutscher Wetterdienst
```

Comment:
Is this a requirement from your employer? What should be the policy for that? add it if you want to? Provide the employer at the date of first commit?

Reply:
I'm not sure if anybody who notices cares, or if anybody who cares notices.  But the way I read our guidelines, we should explicitly mention Deutscher Wetterdienst as having contributed, which I think is reasonable (after all, we contribute during work time).  I think affiliation within AUTHORS.md is a reasonable way to do this, but I'm open to other suggestions.

Reply:
I agree. As long as contributions have been done while at one affiliation only, no dates are required I think. When people change affiliation, one should add start and end dates. So everytime you change affiliation a new line will be added to the authors list. That would be my suggestion.

Reply:
Sounds good to me.